v/vanilla_bean/parameters.vh

```diff
@@ -181,6 +182,8 @@
 `define RV32_FLW_S `RV32_Itype(`RV32_LOAD_FP, 3'b010)
 `define RV32_FSW_S `RV32_Stype(`RV32_STORE_FP, 3'b010)
 
+`define RV32_FRRM {12'b10, 5'b00000, `RV32_CSRRS_FUN3, 5'b?????, `RV32_SYSTEM}
```

Comment:
Would it be possible to define "csr" instruction class like this below, to be consistent with the style of our overall instruction decoding macros?
https://github.com/bespoke-silicon-group/bsg_manycore/blob/7c4059cee15e101f2455282d40c7ab7fad244a7b/v/vanilla_bean/parameters.vh#L70-L74